Tools/CISupport/ews-build/steps.py

```diff
@@ -185,6 +185,19 @@ def _is_hash_outdated(self, pr_json):
         return 0 if pr_sha == self.getProperty('github.head.sha', '?') else 1
 
 
+class ShellMixin(object):
+    def shell_command(self, command):
+        platform = self.getProperty('platform', '*')
+        if platform == 'wincairo':
+            return ['cmd', command]
+        return ['/bin/sh', '-c', command]
+
+    def shell_exit_0(self):
+        platform = self.getProperty('platform', '*')
+        if platform == 'wincairo':
+            return ' && exit 0'
```

Comment:
This should probably also be `||`

Reply:
Actually given the naming of things I don't think this should have any conditional operators it should just return the command